### PR TITLE
Readme setup

### PR DESCRIPTION
**Pull request by PoloSwelsenIlionx**

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -1,12 +1,101 @@
...
+## Install
 
-### Running this example
+### iOS
 
-To run the provided example, you can use `npm start` command.
+```bash
+npm install
+npm run build && npx cap sync
+cd ios && pod install && cd ../
```
Comment (JDouven) [inline example/README.md]: `npx cap sync` also executes `pod install`. Maybe run `npx cap sync ios`?

File: example/README.md [hunk 2]
```diff
@@ -1,12 +1,101 @@
...
+npx cap open ios
+```
+
+### Android
 
 ```bash
-npm start
+npm install
+npm run build && npx cap sync
```
Comment (JDouven) [inline example/README.md]: Maybe run `npx cap sync android`?

File: example/README.md [hunk 3]
```diff
@@ -1,12 +1,101 @@
...
+npx cap open android
 ```
+
+as you can see, two capacitor plugin's are installed of which one the `onfido-capacitor` plugin.
```
Comment (JDouven) [inline example/README.md]:
```suggestion
as you can see, two capacitor plugins are installed of which one the `onfido-capacitor` plugin.
```

**Review by JDouven — CHANGES_REQUESTED** [review]

File: example/README.md (hunk 1 above)
Comment (PoloSwelsenIlionx) [reply example/README.md]: Good one, will solve that!

Comment (EKashpersky) [inline example/README.md]: npx cap sync does this line for iOS

**Review by EKashpersky — APPROVED** [review]

**Review by JDouven — APPROVED** [review]

**Pull request merged**